Sample/SextantSample.UWP/AppBootstrapper.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ReactiveUI;
+using Sextant;
+using Sextant.UWP;
+using SextantSample.Core.ViewModels;
+using SextantSample.UWP.Views;
+using Splat;
+using static Sextant.Sextant;
+
+namespace SextantSample.UWP
+{
+    public class AppBootstrapper : ReactiveObject, IScreen
```

Comment:
Probably better to do this in the App.xaml.cs like the other sample app does.

Reply:
Yes, you're right.